README.md

```diff
@@ -38,9 +38,13 @@ Martha Palmer, Rajesh Bhatt, Bhuvana Narasimhan, Owen Rambow, Dipti Misra Sharma
 }
 </pre>
 
-
 # Changelog
 
+* 2024-1-15 v2.15
+  * Construction annotations in the [UCxn](https://github.com/LeonieWeissweiler/UCxn) framework added to MISC
+  * Verbal lemma is infinitive instead of stem.
```

Comment:
Is this line in the wrong place? I don't see how it relates to UCxn.

Reply:
It is an unintentional copy of previous change in the log. I decided that it would be faster to fix it manually after merge than complaining about the PR.

Reply:
Sorry about that!